sopel/bot.py

```diff
@@ -263,16 +295,22 @@ def register(self, callables, jobs, shutdowns, urls):
             self.register_url_callback(func.url_regex, func)
 
     def part(self, channel, msg=None):
-        """Part a channel."""
+        """Leave a channel.
+
+        :param str channel: the channel to leave
+        :param str msg: the message to display when leaving a channel
+        """
         self.write(['PART', channel], msg)
 
     def join(self, channel, password=None):
-        """Join a channel
+        """Join a channel.
+
+        :param str channel: the channel to join
+        :param str password: an optional channel password
 
         If `channel` contains a space, and no `password` is given, the space is
```

Comment:
Figured it out. Single-backtick gives us `<cite>text</cite>`, which is grossly incorrect for these uses both semantically and visually (it renders in italicized prose font). Using double-backticks yields (I'm simplifying) `<code>text</code>`, which is what we want (renders as monospace).

```suggestion
        If ``channel`` contains a space, and no ``password`` is given, the space is
```